client/src/components/NavDrawerCloseContext.tsx

```diff
@@ -0,0 +1,7 @@
+import { createContext } from 'react';
+
+export const closeContext = createContext({
+  isItOpen: true || false,
```

Comment:
`true || false` is odd. It reads as "evaluate to `true`, if `true` is truthy, otherwise evaluate to `false`." But `true` is always truthy, so this expression is always `true`.

So you could just say `isItOpen: true,`.